src/components/CroppingOverlay.js

```diff
@@ -63,8 +63,10 @@ const useStyles = makeStyles(() => ({
   dialogButton: {
     backgroundColor: "rgba(255,255,255,0.8) !important",
     borderRadius: "25%",
-    left: ({ buttonOutside }) => (buttonOutside ? "-35px" : "5px"),
+    transform: ({ buttonOutside }) =>
+      buttonOutside ? "translateX(calc(-100% - 5px))" : "translate(5px)",
```

Comment:
Wäre hier nicht in beiden Fällen ein translateX` konsequenter?

Reply:
Oh ja, ich dachte, ich hätte beide geändert. Fix kommt.